packages/ionq/tests/ionq/circuit/transpile/test_ionq_native_transpiler.py

```diff
@@ -97,12 +97,18 @@ def ionq_circuit_state(
     qs_circuit = qulacs.QuantumCircuit(circuit.qubit_count)
     for gate in circuit.gates:
         if gate.name in ionq_gates_1:
+            # the second argument of `.add_dense_matrix_gate()`
+            # takes in NDArray[complex128, _Shape]
+            # (_Shape should be the first argument)
             qs_circuit.add_dense_matrix_gate(
-                gate.target_indices[0], ionq_gate_matrix(gate)
+                gate.target_indices[0], ionq_gate_matrix(gate)  # type: ignore
             )
         elif gate.name in ionq_gates_2:
+            # the second argument of `.add_dense_matrix_gate()`
+            # takes in NDArray[complex128, _Shape]
+            # (_Shape should be the first argument)
```

Comment:
I think the comment in `qulacs/circuit/__init__.py` is clearer:

```suggestion
            # We need to disable type check due to an error in qulacs type annotation
            # https://github.com/qulacs/qulacs/issues/537
```